checkbox.js

```diff
@@ -0,0 +1,125 @@
+/*jshint esversion: 6*/
+/*global document*/
+/*global alert*/
+
+class Data {
+    static getCheckboxList() {
+        return ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday', 'None'];
+      } 
+  }
+  
+  class CheckboxOperations {
+    
+    createCheckboxList() {
+      let ul = this.createUlElementForCheckbox();
+      document.body.appendChild(ul);
+        this.createAndAppendCheckboxListTo(ul);
```

Comment:
When appending elements, read about document fragment. It is less costly and more robust